package.json

```diff
@@ -249,5 +249,11 @@
   "license": "MIT",
   "optionalDependencies": {
     "fsevents": "*"
+  },
+  "resolutions": {
+    "relay-compiler": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/relay-compiler-1.5.0-plugin.2.tgz",
+    "relay-runtime": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/relay-runtime-1.5.0-plugin.2.tgz",
+    "babel-plugin-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/babel-plugin-relay-1.5.0-plugin.2.tgz",
+    "react-relay": "https://github.com/alloy/relay/releases/download/v1.5.0-plugin.2/react-relay-1.5.0-plugin.2.tgz"
```

Comment:
This should be `artsy.3`, I think you should just copy verbatim the entry from Reaction https://github.com/artsy/reaction/blob/80f4d9802fd090d943df84470fc395dbce8aa824/package.json#L47